### DISCUSSION TITLE
Display all devices of a certain grouping at a glance

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#64] [Ideas]
Devices can be grouped in a wide variety of constellations across device models, types and other species in everyday clinical practice.

Currently, you can combine filters, but often you do not get the desired result directly.
Another option is to create groups. But the devices are grouped according to models. After that, there is no way to break down this grouping to individual devices.

The group function is actually perfect. There should only be a possibility to display all individual devices/Inventories of the device model in a group at a glance.

**Comment by YvesR** [comment]
Is this covered with the new upcoming device-systems feature or a different use case still?
